### PR TITLE
Lk86/spot vm integration develop

### PR DESCRIPTION
**Pull request by lk86**
This is the same as #12076, to allow integration tests to pass, this time against `develop`.

### CODE REVIEW

**Comment by lk86** [comment]
!ci-build-me

**Comment by lk86** [comment]
!ci-build-me

**Review by MartinMinkov — APPROVED** [review]

File: automation/terraform/modules/testnet-alerts/templates/testnet-alert-rules.yml.tpl [hunk 1]
```diff
@@ -244,6 +244,18 @@ groups:
       description: "{{ $value }} Missing block count is critically high on network {{ $labels.testnet }}."
       runbook: "https://www.notion.so/minaprotocol/Archive-Node-Metrics-9edf9c51dd344f1fbf6722082a2e2465"
 
+  - alert: FewBlocksPerHour
+    expr: quantile by (testnet) (0.5, increase(Coda_Transition_frontier_max_blocklength_observed ${rule_filter} [30m])) < 1
+    for: ${alert_evaluation_duration}
+    labels:
+      testnet: "{{ $labels.testnet }}"
+      severity: warning
```
Comment (mrmr1993) [inline automation/terraform/modules/testnet-alerts/templates/testnet-alert-rules.yml.tpl:252]: Should this be `critical`?

**Comment by lk86** [comment]
!ci-build-me

**Comment by lk86** [comment]
!ci-build-me

**Comment by lk86** [comment]
!ci-build-me

**Comment by lk86** [comment]
!ci-build-me

**Comment by lk86** [comment]
!ci-build-me

**Comment by lk86** [comment]
!ci-build-me

**Comment by lk86** [comment]
!ci-build-me

**Comment by lk86** [comment]
!ci-build-me

**Comment by lk86** [comment]
!ci-build-me

**Comment by lk86** [comment]
!ci-build-me

**Comment by lk86** [comment]
!ci-build-me

**Pull request merged**